test/behavior/align.zig

```diff
@@ -1,21 +1,22 @@
 const std = @import("std");
 const expect = std.testing.expect;
+const expectEqual = std.testing.expectEqual;
 const builtin = @import("builtin");
 const native_arch = builtin.target.cpu.arch;
 
 var foo: u8 align(4) = 100;
 
 test "global variable alignment" {
-    comptime try expect(@typeInfo(@TypeOf(&foo)).Pointer.alignment == 4);
-    comptime try expect(@TypeOf(&foo) == *align(4) u8);
+    comptime try expectEqual(@typeInfo(@TypeOf(&foo)).Pointer.alignment, 4);
```

Comment:
These arguments are backwards. The expected value is on the left.

Reply:
Maybe I should close this PR and redo when #4437 concludes?